gulpfile.js

```diff
@@ -8,22 +8,41 @@ var fail   = require('gulp-fail');
 var flatten = require('gulp-flatten');
 var runSequence = require('gulp-run-sequence');
 var clean = require('gulp-clean');
+var rename = require('gulp-rename');
 
 var lambdaOptions = {
     region: 'eu-west-1'
 };
 
+var env = (function() {
+  function prod(str) { return /prod/gi.test(str); }
+
+  return process.argv.some(prod) ? 'PROD' : 'CODE';
+})();
+
+var envConfig = 'email-signup-config-' + env + '.js';
```

Comment:
Isn't this going to throw if it doesn't exist, meaning we can't run `downloadCredentials`?